app/src/main/java/com/babylon/wallet/android/presentation/settings/personas/personadetail/PersonaDetailScreen.kt

```diff
@@ -129,7 +129,15 @@ private fun PersonaDetailContent(
                 RadixCenteredTopAppBar(
                     title = state.persona?.displayName?.value.orEmpty(),
                     onBackClick = onBackClick,
-                    windowInsets = WindowInsets.statusBars
+                    windowInsets = WindowInsets.statusBars,
+                    actions = {
+                        if (state.persona != null) {
+                            UnderlineTextButton(
+                                text = stringResource(id = R.string.authorizedDapps_personaDetails_editPersona),
```

Comment:
in zeplin I see this as `Edit`, are you sure it should be "Edit Persona"?

Reply:
You're right, it should be "Edit", initially I edited this string in the localization repo but now I see it's being used in another place as well. I'll add a new string.